### PR TITLE
Fix EZP-24394: Implement FieldDefinition form mapper for eztime

### PR DESCRIPTION
**Pull request by glye**
Manual testing. This is based on ezdatetime, but thankfully the tricky bits of that are not needed here, so it's pretty simple.

https://jira.ez.no/browse/EZP-24394

### CODE REVIEW

**Comment by glye** [comment]
Fixed translation copypasta error.

**Comment by pspanja** [comment]
+1

File: lib/FieldType/Mapper/TimeFormMapper.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                    'required' => false,
+                    'property_path' => 'fieldSettings[useSeconds]',
+                    'label' => 'field_definition.eztime.use_seconds',
+                ]
+            )
+            ->add(
+                'defaultType',
+                'choice',
+                [
+                    'choices' => array(
```
Comment (pspanja) [inline lib/FieldType/Mapper/TimeFormMapper.php]: (We should probably be consistent with short syntax.)

**Comment by glye** [comment]
Short array syntax everywhere.

**Comment by glye** [comment]
Rebased after ezcountry commit

**Comment by lolautruche** [comment]
+1

**Comment by bdunogier** [comment]
+1

**Pull request merged**